app-shells/squirrelsh/files/squirrelsh-1.2.7-gcc6.patch

```diff
@@ -0,0 +1,15 @@
+--- a/shell/base.cpp
++++ b/shell/base.cpp
+@@ -14,10 +14,11 @@
+ // You should have received a copy of the GNU General Public License
+ // along with this program. If not, see <http://www.gnu.org/licenses/>.
+ 
+-#include "common.h"
+ #include <string.h>
+ #include <string>
+ 
++#include "common.h"
```

Comment:
from your report, would it be possible to fix this **properly** once and for all? I never use math.h in C++, use fully qualified std::min and std::max everywhere? don't see this as a requirement, it would just be a nice thing for the ecosystem, as it would make the codebase more robust, and less flaky.